### DISCUSSION TITLE
Новые страны для мира

### DISCUSSION THREAD

**Discussion by Vlados-Ddos** [voxeltycoon/issues#702] [General]
Здраствуйте! Я хотел бы вам предложить добавить новую страну для генерации названий городов в игре - **Беларусь**. Я надеюсь что вы это рассмотрите)

**Comment by victorleblais** [comment]
Please write your suggestions in English.

**Reply by Vlados-Ddos** [reply]
Hello there! I would like to suggest that you add a new country to generate city names in the game - Belarus. I hope you will consider this)
